js_modules/dagit/packages/core/src/partitions/PartitionRunMatrix.tsx

```diff
@@ -72,14 +72,14 @@ interface PartitionRunMatrixProps {
   pipelineName: string;
   partitions: PartitionRuns[];
   repoAddress: RepoAddress;
-  runTags: TokenizingFieldValue[];
-  setRunTags: (val: TokenizingFieldValue[]) => void;
+  runFilters: RunFilterToken[];
+  setRunFilters: (val: RunFilterToken[]) => void;
   stepQuery: string;
   setStepQuery: (val: string) => void;
 }
 
-const _backfillIdFromTags = (runTags: TokenizingFieldValue[]) => {
-  const [backfillId] = runTags
+const _backfillIdFromTags = (runFilters: RunFilterToken[]) => {
+  const [backfillId] = runFilters
     .filter((_) => _.token === 'tag' && _.value.startsWith('dagster/backfill='))
```

Comment:
strange..I've only ever seen the _ syntax for a parameter if you deliberately were not consuming the parameter. Seems that just doing ({token, value}) would've been better if you didnt want to name the obj but 🤷‍♂️